### PR TITLE
disable non-prep plan cache

### PR DESCRIPTION
**Pull request by chrysan**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Non-prepared plan cache is an experimental feature in v7.1.0.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s): https://github.com/pingcap/docs-cn/pull/13662

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -1323,12 +1323,15 @@ MPP 是 TiFlash 引擎提供的分布式计算框架，允许节点之间的数
...
+
 - 作用域：SESSION | GLOBAL
 - 是否持久化到集群：是
 - 类型：布尔型
-- 默认值：在 v7.1.0 之前版本中为 `OFF`，即默认关闭。在 v7.1.0 及之后的版本中为 `ON`，即默认开启。
+- 默认值：`OFF`
 - 这个变量用来控制是否开启[非 Prepare 语句执行计划缓存](/sql-non-prepared-plan-cache.md)。
-- 从旧版本升级到 v7.1.0 及之后版本，该变量开关保持升级前的状态。
 
 ### `tidb_enable_non_prepared_plan_cache_for_dml` <span class="version-mark">从 v7.1.0 版本开始引入</span>
```
Comment (Oreoxmt) [inline system-variables.md:1337]:
```suggestion

> **警告：**
>
> 非 Prepare 语句执行计划缓存目前为实验特性，不建议在生产环境中使用。该功能可能会在未事先通知的情况下发生变化或删除。如果发现 bug，请在 GitHub 上提 [issue](https://github.com/pingcap/tidb/issues) 反馈。

```
tidb_enable_non_prepared_plan_cache_for_dml 也与 non-prepared plan cache 相关，建议添加实验特性说明

**Review by Oreoxmt — APPROVED** [review]
Rest LGTM

**Comment by Oreoxmt** [comment]
@qw4990 @songrijie PTAL

**Review by songrijie — APPROVED** [review]

**Review by qw4990 — APPROVED** [review]
LGTM

**Review by hfxsd — APPROVED** [review]

**Pull request merged**